liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy

```diff
@@ -112,12 +112,20 @@ grant execute on any_procedure_name to ANY_USER3/
 """
 
     private static final String INVALID_CHANGELOG = "select * from table1"
-    private static final String INVALID_CHANGELOG_INVALID_PRECONDITION = "--liquibase formatted sql\n" +
+    private static final String INVALID_CHANGELOG_INVALID_PRECONDITION =
+            "--liquibase formatted sql\n" +
             "\n" +
             "--changeset bboisvert:invalid_precondition\n" +
             "--precondition-invalid-type 123\n" +
             "select 1;"
 
+    private static final String INVALID_CHANGELOG_INVALID_PRECONDITION_PATTERN =
+            "--liquibase formatted sql\n" +
+                    "\n" +
+                    "--changeset bboisvert:invalid_precondition\n" +
+                    "--precondition-invalid-type 123\n" +
+                    "select 1;"
```

Comment:
This looks identical to INVALID_CHANGELOG_INVALID_PRECONDITION. Do we need both?